src/NUnitFramework/tests/Attributes/ApartmentAttributeTests.cs

```diff
@@ -24,17 +24,24 @@
 #if APARTMENT_STATE
 using System;
 using System.Threading;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using NUnit.TestData;
+using NUnit.TestUtilities;
 
 namespace NUnit.Framework.Attributes
 {
     [TestFixture]
     public class ApartmentAttributeTests : ThreadingTests
     {
+#if APARTMENT_STATE
         [Test]
-        public void ApartmentStateUnknownThrowsException()
+        public void ApartmentStateUnknownIsNotRunnable()
         {
-            Assert.That(() => new ApartmentAttribute(ApartmentState.Unknown), Throws.ArgumentException);
+            var testSuite = TestBuilder.MakeFixture(typeof(ApartmentDataRequiresThreadAttribute));
```

Comment:
Isn't this the wrong type?
```suggestion
            var testSuite = TestBuilder.MakeFixture(typeof(ApartmentDataApartmentAttribute));
```

Reply:
Ugh - good spot! Thanks!